push-ci-build.sh

```diff
@@ -159,14 +156,9 @@ if ((FLAGS_federation)); then
   ############################################################################
   common::stepheader PUSH FEDERATION
   ############################################################################
-  if ((FLAGS_nomock)); then
-    logecho -n "Push federation images: "
-    # Because this executes outside the scope and control of the release infra
-    # we can't provide a true mock, so just print with $LOGRUN_MOCK
-    logrun -s $LOGRUN_MOCK ${KUBE_ROOT}/build/push-federation-images.sh
-  else
-    logecho "FEDERATION Image push skipped for mocked workflow..."
-  fi
+  logecho -n "Push federation images: "
+  # FEDERATION_PUSH_REPO_BASE should be set by the calling job (yaml)
```

Comment:
Yes, it is required for calling `push-federation-images.sh->push-federation-images()`.  There are so many layers here it seems useful to leave some breadcrumbs.